specification/modsimworkbench/resource-manager/Microsoft.ModSimWorkbench/preview/2021-03-01-preview/connectors.json

```diff
@@ -0,0 +1,909 @@
+        ],
+        "description": "Add license to an ModSim Workbench connector.",
+        "operationId": "connectors_LicenseAdd",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "in": "path",
+            "name": "resourceGroup",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group"
+          },
+          {
+            "in": "path",
+            "name": "workbenchName",
+            "required": true,
+            "type": "string",
+            "description": "The name of the workbench"
+          },
+          {
+            "name": "chamberName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the chamber."
+          },
+          {
+            "in": "path",
+            "name": "name",
+            "required": true,
+            "type": "string",
+            "description": "The name of the connector."
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "description": "License key parameters",
+            "schema": {
+              "$ref": "#/definitions/LicenseKeyParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default Error Response",
+            "schema": {
+              "$ref": "./modsimworkbenches.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Connectors License Add": {
+            "$ref": "./examples/ConnectorsLicenseAdd.json"
+          }
+        },
+        "security": [
+          {
+            "azure_auth": [
+              "user_impersonation"
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "RemoteDesktop": {
+      "type": "object",
+      "description": "Remote Desktop properties.",
+      "properties": {
+        "etx": {
+          "$ref": "#/definitions/Etx",
+          "description": "The etx object."
+        }
+      }
+    },
+    "Etx": {
```

Comment:
>Etx [](http://example.com/codeflow?start=5&length=3)

If not everyone will know what an 'Etx' is, you should consider using a longer name.

Reply:
ETX and OTDS are third party trademarks and although I can update the description on the comments above the preference for the property would still be to keep it as ETX

Reply:
I think we want to avoid using trademarks in property names in most cases, but maybe this case is kind of special, because you are defining types which will work only for that 3rd party solution?